docs/tutorial/index.md

```diff
@@ -209,12 +204,20 @@ Now, ensure the client has access to your cloud (i.e., knows where to find your
 :host: my-juju-vm
 :input: juju clouds --client
 
-# Output should show microk8s.
-# If it doesn't: Exit the VM (`exit`), re-enter it (`multipass shell my-juju-vm`), then try again.
+Only clouds with registered credentials are shown.
+There are more clouds, use --all to see them.
+You can bootstrap a new controller using one of these clouds...
+
+Clouds available on the client:
+Cloud      Regions  Default    Type  Credentials  Source    Description
+localhost  1        localhost  lxd   0            built-in  LXD Container Hypervisor
+microk8s   1        localhost  k8s   1            built-in  A Kubernetes Cluster
 
 ```
 
-Ensure also that the client has access to Charmhub by launching a random query, e.g., "ingress", and asking for more information about one of the results it shows:
+(If this doesn't show any output: Exit the VM (`exit`), re-enter it (`multipass shell my-juju-vm`), then try again.)
```

Comment:
This is why I'm not really a fan of using multipass in the tutorial. We're now constantly intermixing the tutorial with - how to use multipass, or use the right window.